### ISSUE TITLE
Informix CURRENT_TIMESTAMP emulation doesn't work as DDL DEFAULT expression

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#14316]
This is valid in Informix:

```sql
SELECT CURRENT
```

But this isn't:

```sql
CREATE TABLE x (y DATETIME YEAR TO FRACTION (5) DEFAULT CURRENT);
```

Despite the datetime precision being clear from context, we have to supply it explicitly, as `CURRENT YEAR TO FRACTION (5)`

**Comment by lukaseder** [comment]
This was discovered as part of https://github.com/jOOQ/jOOQ/issues/14313

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.18.0, 3.17.6 (https://github.com/jOOQ/jOOQ/issues/14317), and 3.16.12 (https://github.com/jOOQ/jOOQ/issues/14318)
